.gitlab-ci.yml

```diff
@@ -3087,6 +3081,32 @@ docker_build_dogstatsd_amd64:
     IMAGE: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent/dogstatsd
     BUILD_CONTEXT: Dockerfiles/dogstatsd/alpine
 
+#
+# Internal image deployment
+#
+docker_trigger_internal_amd64:
+  stage: image_deploy
+  rules:
+    - <<: *if_version_7
+      when: on_success
+    - <<: *if_deploy_7
+      when: on_success
+  needs:
+    - job: docker_build_agent7_jmx
+      artifacts: false
+  trigger:
+    project: DataDog/images
+    branch: master
+    strategy: depend
+  variables:
+    IMAGE_VERSION: tmpl-v1
+    IMAGE_NAME: datadog-agent
+    RELEASE_TAG: $CI_COMMIT_REF_SLUG
+    BUILD_TAG: $CI_COMMIT_REF_SLUG
+    TMPL_AGENT_SRC_IMAGE: v$CI_PIPELINE_ID-${CI_COMMIT_SHORT_SHA}-7-jmx-amd64
+    RELEASE_STAGING: "true"
+    RELEASE_PROD: "true"
```

Comment:
I think since this job is only available on the "master" branch. it seems ok to push in both like you did.

Or, should we have 2 jobs. on for staging, the other one for prod?

Reply:
It's only available on tags, so I think we want to have each tag pushed to both places.